tests/testthat/test-pivot-wide.R

```diff
@@ -65,6 +65,16 @@ test_that("data frame columns pivot correctly", {
   expect_equal(out$b$y, 7:8)
 })
 
+test_that("works with data.table and empty key_vars", {
+  skip_if_not_installed("data.table")
```

Comment:
We assume that suggested packages are installed for development, so you can remove this.